src/pages/ContactForm/DropdownComponent.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+  import { StyleSheet } from 'react-native';
+  import { Dropdown } from 'react-native-element-dropdown';
+  import AntDesign from '@expo/vector-icons/AntDesign';
+
+  const data = [
+    { label: 'Item 1', value: '1' },
+    { label: 'Item 2', value: '2' },
+    { label: 'Item 3', value: '3' },
+  ];
+
+  const DropdownComponent = () => {
+    const [value, setValue] = useState(null);
+
+    return (
+      <Dropdown
```

Comment:
paddingLeft property of placeholderStyle should be 20.
placeholderStyle={{ paddingLeft: 20 }}